### PR TITLE
THEMES-1122: Update primary and secondary fonts

### PR DESCRIPTION
**Pull request by vgalatro**
## Description

This PR updates the font families in storybook and in the alias tokens `news.json` to 'Inter' and 'Lora'

## Jira Ticket

- [THEMES-1122](https://arcpublishing.atlassian.net/browse/THEMES-1122)

## Acceptance Criteria

In v2:

- primary-font default is Inter
- secondary-font default is Lora
- Updated in storybook

## Test Steps

1. Checkout this branch `git checkout THEMES-1122`
2. Run storybook `npm run storybook`
3. Check some of the blocks, ensure that the primary font is 'Inter' and the secondary font is 'Lora'.

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-1122]: https://arcpublishing.atlassian.net/browse/THEMES-1122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by blakeganderson — APPROVED** [review]
LGTM (pending acceptance of chromatic UI changes)

**Comment by pardistaherzadeh** [comment]
It is approved and ready to be merged. @vgalatro

**Comment by vgalatro** [comment]
@pardistaherzadeh , I merged in the latest from `arc-themes-release-version-2.0.3` and a few more changes showed up needing review. It looks related to the fix for images that was recently merged in. Also there's a related PR in `arc-themes-components`, https://github.com/WPMedia/arc-themes-components/pull/184, that needs an approval as well. Thanks for looking into these!

**Pull request merged**
